Makefile

```diff
@@ -9,7 +9,7 @@ build-react:
 	cd react-display && npm install && npm run build
 
 build-go:
-	go build -o out/discombobulator cmd/discombobulator/main.go
+	go build -o out/scale-simulator cmd/scale-simulator/main.go
```

Comment:
was fun while it lasted :cry:

Reply:
haha, we can reuse that name for something more appropriate down the line. I'm sure it'll come up.